spyder/plugins/explorer/widgets/explorer.py

```diff
@@ -1681,9 +1694,17 @@ def __init__(self, parent=None, name_filters=['*.py', '*.pyw'],
         button_previous = QToolButton(self)
         button_next = QToolButton(self)
         button_parent = QToolButton(self)
+        self.filter_on = False
+        filter_button = create_toolbutton(
+            self, icon=ima.icon("filter"),
+            tip=_("Activate/deactivate filter"),

```

Comment:
```suggestion
            tip=_("Activate/deactivate filename filters"),
```

Since we have plenty of space, I suggest the tooltip read `Activate/deactivate filename filters` to be a little more descriptive about what it does.